chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomsReceivedIntegrationTest.kt

```diff
@@ -91,8 +95,8 @@ internal object JoinedRoomsReceivedIntegrationTest : Spek({
             }
 
             it("then no actions are dispatched") {
-                verify(exactly = 0) {
-                    dispatcher(any())
+                verify(exactly = 1) {
```

Comment:
I suspect this invalidates the test description

Reply:
👍 so i was undecided on this one - none of the Subscription actions should be fired. Have updated the description to say `then only CurrentUserReceived is dispatched`

Reply:
this test is for testing stuff only around joined rooms, let's remove any verification around current user. here it should be precisely verifying that the dispatcher has not been called with any joined room related actions, i.e. 4 lines for 4 relevant events in the context of this test

Reply:
Oops, sorry, I didn't read around the context with of the test, I just saw the 0 change to 1. Agree with Marek.

Reply:
Hmm. This part is hard. The tests want to ensure *only* the expected actions are called (nothing extra). To do that I use `confirmVerified(dispatcher)` - however they'll all fail if I don't say we expect the currentUser to be updated too. What's the best way of moving forward with this?

Reply:
We resolved on slack to remove the confirmVerified call and we're okay on the risk that other actions might get called that we don't know about.